### DISCUSSION TITLE
Adding new filter conditions greater/less (> or <) for a self created field customber number

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#16998] [How to] [answered]
Hi,

I need a filter / condition for a self created field "customer number".

Each organization/customer/client has his own customer-number in the system. I would like to create a filter/condition, that shows me all numbers **above / greater than field >** 5000 for example. Or a condition with **less than <** 5000.

I think the problem is, that when I first created the extra customer-number field I accidently choosed type "varchar" and later transformed it into a "int(10)" field directly in the mysql database. So now for this field I don't have the possibility to choose > or < as a condition, but I can see that the condition is available for other "int" fields. The "extra fields module" in YF also still shows me that the field is a "Text (UiType: 1)" that is not any more - so I think this is the reason why I can't find the conditions needed.

Can someone point me where to change the "Text (UiType: 1)"  directly in the database?

Thanks!

Kind regards

**Comment by rskrzypczak** [comment, marked as answer]
vtiger_field => uitype, typeofdata

**Comment by blu-IT** [comment]
@rskrzypczak 

Wow, thanks! OK, found that in the database. For integer what _uitype_ and _typeofdata_ I have to set?

**Reply by rskrzypczak** [reply]
Check yourself based on another integer field (_7, I~O_)

**Reply by blu-IT** [reply]
OK, checked it with a reference field: uitype "7" and typeofdate "I~O" is correct!

Thanks a lot again!

**Comment by blu-IT** [comment]
OK, there seems still to be something tricky:

Changing the values now shows my custumer numbers like this: 5 001 instead of 5001.

**Reply by blu-IT** [reply]
OK, sorry for that. I had to change the way numeric values are displayed in the personal settings / profile in YF.

**Reply by ArniSamano** [reply]
> OK, sorry for that. I had to change the way numeric values are displayed in the personal settings / profile in YF.

Yes but that will affect whole inventory fields. Number field without separators is necessary
